### DISCUSSION TITLE
posts/python-custom-formatting

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#534] [Announcements]
# posts/python-custom-formatting

This blog post explains how to use Python's formatting mini-language to format strings and datetimes, as well as how to create custom formatting options for any object. It also provides an implementation for formatting geographic coordinates in various formats with different symbol options.

http://127.0.2.2:8080/posts/python-custom-formatting.html

**Comment by MinhTD18** [comment]
a
